workflow/snakemake_rules/export_for_nextstrain.smk

```diff
@@ -135,6 +137,11 @@ rule dated_json:
         dated_tip_frequencies_json = "auspice/{prefix}_{build_name}_{date}_tip-frequencies.json"
     benchmark:
         "benchmarks/dated_json_{prefix}_{build_name}_{date}.txt"
+    wildcard_constraints:
+        # Allow build names to contain alphanumeric characters, underscores, and hyphens
+        # but not special strings used for Nextstrain builds.
+        build_name = r'(?:[a-zA-Z0-9-_](?!(tip-frequencies|\d{4}-\d{2}-\d{2})))+',
+        date = r"[0-9][0-9][0-9][0-9]-[0-9][0-9]-[0-9][0-9]"

```

Comment:
Can we standardize between `\d{4}-\d{2}-\d{2}` and `[0-9][0-9][0-9][0-9]-[0-9][0-9]-[0-9][0-9]` here? They're equivalent regex patterns, would be nice to keep consistent.

Reply:
Oh my! Strong vote for the `\d{4}-\d{2}-\d{2}` pattern 😂

Reply:
Agreed! Yes, I'll pick the less verbose option.

Reply:
Fixed in 1b4beda

Reply:
I see, maybe another option would be `[0-9]{4}-[0-9]{2}-[0-9]{2}`?

Reply:
Yep!